codelabs/Utopia Wholesale Codelab/3-verifier.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Verifier
+sidebar_position: 3
+---
+
+# Verifier
+
+
+Multipaz Identity Verifier( Multipaz Identity Reader) allows you to request and display identity credentials from another person using QR codes, NFC, or Bluetooth Low Energy (BLE), in compliance with ISO/IEC 18013-5:2021. It works entirely offline, without requiring an Internet connection, and supports importing IACA certificates and VICALs.Click **[here](https://apps.multipaz.org/identityreader/identity-reader-0.3.0-pre.7.bdf1bce.apk)** to download the latest APK for sideloading.
+
+The source code for the Multipaz Identity Reader is available **[here](https://github.com/davidz25/MpzIdentityReader)**.
+
+
+# How to Import a Certificate
+
+The **Multipaz Identity Verifier** requires the issuer’s certificate to verify credentials. Only the verifier app needs this certificate. The issuer can be part of the trusted list.
+
+Follow these steps to import the certificate:
+
+1. Open the **Holder** app project.
+2. Go to the folder:  
+   `/composeApp/src/commonMain/composeResources`
+3. Find the file:  
+   `iaca_certificate.pem`
+4. Copy `iaca_certificate.pem` to your phone.
+5. On your phone, open the **Multipaz Identity Verifier** app.
+6. Tap the **hamburger menu** (☰) in the top-left corner.
+7. Navigate to: **Settings** → **Trusted Issuers**
+8. Tap the **"+" button** (bottom right).
+9. Choose **Import Certificate** and select the `iaca_certificate.pem` file.
+
+ You’re now ready to use the Verifier app to scan a barcode from the Holder app.
+
+---
+
+# How to Generate a Certificate (Optional)
```

Comment:
I think this section might be better placed in the Holder app, since that’s where we define the IACA. Would you mind moving it there?